File: src/day38_Inheritance/carTask/Tesla.java
package day38_Inheritance.carTask;

public class Tesla extends car {

    public Tesla(String brand, String model, int year, double price, String color, int miles) {
        super(brand, model, year, price, color, miles);
    }

    public void autoPilot(){
        System.out.println(brand+" "+model+" in autopilot mode");
    }


    /*
    @Override
    public void start() {
        System.out.println("Say \"Start\" to start "+brand+" "+model);
    }

     */
}

File: src/Day38_Inheritance/carTask/Tesla.kt
package Day38_Inheritance.carTask

class Tesla(brand: String, model: String, year: Int, price: Double, color: String, miles: Int) : Car(brand, model, year, price, color, miles) {
    fun autoPilot() {
        println("$brand $model in autopilot mode")
    }

    override fun start() {
        println("Say \"Start\" to start $brand $model")
    }
}